### DISCUSSION TITLE
Send message (text or vocal) to everyone in the area

### DISCUSSION THREAD

**Discussion by gparant** [thecodingmachine/workadventure#3436] [WorkAdventure’s Roadmap]
We want to improve the virtual experience by offering features that represent possible interactions in real life.

Imagine that you arrive at your office or on a bench in your open space and you want to say "hello!" to your colleagues. On WorkAdventure, you should take the time to move around and see each person one by one.

Another need, you want to share an idea, have a help on document, … Today on WorkAdventure you must target a person or ask to come, by going to see your colleagues one hand one, in a place to be able to exchange easier.

This feature will allow you to share a topic of discussion and conversation. The topic would then be visible to everyone and users could decide to participate and join you or leave the topic aside because they do not wish to take part of the discussion.

Don’t hesitate to leave us a comment or share your feeling.

See you soon on [WorkAdventure](https://play.staging.workadventu.re/@/tcm/workadventure/wa-village) 🚀

**Comment by highno** [comment]
Users at my place have asked for such a feature too. They want to have some kind of visible Help-Topic that they can carry around. (aka visible below your name) That way they are requesting everyone to help with that topic if they can.
They would not want to text everyone or make a call to the room. I understand that feature is nice, too. But the way my users ask for is less disturbing for everyone, yet not that powerful.
